# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/mac-fdisk/install-man.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mac-fdisk-0.1.orig/Makefile.orig	2003-11-26 08:04:05.000000000 +0100
+++ mac-fdisk-0.1.orig/Makefile	2003-11-26 08:05:52.000000000 +0100
@@ -7,7 +7,7 @@
 clean:
 	rm -f *.o pdisk fdisk mac-fdisk pmac-fdisk
 
-install: pdisk fdisk
+install: pdisk fdisk install-man
 	-rm -f pmac-fdisk mac-fdisk
 	ln pdisk mac-fdisk
 	ln fdisk pmac-fdisk
@@ -29,3 +29,7 @@
 partition_map.h: dpme.h
 dpme.h: bitfield.h
 
+install-man:
+	install -m 644 mac-fdisk.8.in ${DESTDIR}/usr/share/man/man8/
+	install -m 644 pmac-fdisk.8.in ${DESTDIR}/usr/share/man/man8/
+
